python/mlc_chat/serve/entrypoints/openai_entrypoints.py

```diff
@@ -92,6 +103,10 @@ async def completion_stream_generator() -> AsyncGenerator[str, None]:
                 prompt, generation_cfg, request_id
             ):
                 num_completion_tokens += num_delta_tokens
+                if delta_text == "":
```

Comment:
The delta only means insertion without delete, is that right? I wonder can we rename it to `appended_text` instead? (I know it's not introduced in this  PR, so we can adopt the renaming in later PRs.)

Reply:
“Delta” is used frequently I think. Our ChatModule uses the term “delta message,” and for example, OpenAI uses “delta” https://platform.openai.com/docs/api-reference/chat/streaming#chat/streaming-choices (you need to expand the entry “choices”. In this case, `appended_text` seems a bit unclear to me what “appended” means. I think we can go with the term “delta”.